tests/cloud-http-get-test.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+
+load './bats-helpers/bats-support/load'
+load './bats-helpers/bats-assert/load'
+
+setup() {
+    FILE=$(mktemp)
+    cp ./tests/cloud-http-get-test.yaml $FILE
+    UUID=$(uuidgen | awk '{print tolower($0)}')
+    TOPIC=${UUID}-topic
+    CONNECTOR=${UUID}-cloud-http-get-test
+
+    sed -i.BAK "s/CONNECTOR/${CONNECTOR}/g" $FILE
+    sed -i.BAK "s/TOPIC/${TOPIC}/g" $FILE
+    cat $FILE
+
+    fluvio cloud login --email ${FLUVIO_CLOUD_TEST_USERNAME} --password ${FLUVIO_CLOUD_TEST_PASSWORD} --remote 'https://dev.infinyon.cloud'
+    fluvio topic create $TOPIC
+    fluvio cloud connector create --config $FILE
+
+    cargo build -p http-source
+    ./target/debug/http-source --config $FILE & disown
+    CONNECTOR_PID=$!
+}
+
+teardown() {
+    fluvio cloud connector delete $CONNECTOR
+    kill $CONNECTOR_PID
+}
+
+@test "cloud-http-get-test" {
+    echo "Starting consumer on topic $TOPIC"
+    echo "Using connector $CONNECTOR"
+    sleep 13
```

Comment:
The provisioning of the connector on the cloud can take more than 13 seconds, we need more reliable delays/retries here.